service/resource/configmap/current_test.go

```diff
@@ -0,0 +1,94 @@
+package configmap
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/giantswarm/micrologger/microloggertest"
+)
+
+// TestGetCurrentState tests the GetCurrentState method.
+func TestGetCurrentState(t *testing.T) {
```

Comment:
It would be more useful to have a more structured name like `Test_Resource_ConfigMap_GetCurrentState`.